statistics/merge_worker.go

```diff
@@ -47,12 +50,14 @@ type topnStatsMergeWorker struct {
 func NewTopnStatsMergeWorker(
 	taskCh <-chan *TopnStatsMergeTask,
 	respCh chan<- *TopnStatsMergeResponse,
-	wrapper *StatsWrapper) *topnStatsMergeWorker {
+	wrapper *StatsWrapper,
+	killed *uint32) *topnStatsMergeWorker {
 	worker := &topnStatsMergeWorker{
 		taskCh: taskCh,
 		respCh: respCh,
 	}
 	worker.statsWrapper = wrapper
+	worker.killed = killed
```

Comment:
atomic.[LoadUint32](https://cs.opensource.google/go/go/+/go1.19.2:src/sync/atomic/doc.go;l=126)

Reply:
we passed pointer here